runtime_kanto/test/integration/integration_test.py

```diff
@@ -57,13 +58,16 @@ def check_container_is_running(container_name: str) -> bool:
         app_name (str): App name
     """
     return json.loads(
-        subprocess.run(
-            ["kanto-cm", "get", "-n", container_name], stdout=subprocess.PIPE
-        ).stdout.decode("utf-8")
+        subprocess.check_output(
```

Comment:
Any reason to change this? subprocess.run is the recommended way to do it.

Reply:
Reverted.